### PR TITLE
docs: add information about `type` parameter in `README.md`

### PR DESCRIPTION
**Pull request by JKirito**
## Fixes Issue
Added Data about types and their suitable values field to README.md

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/821"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Alternatively, skip all the steps by using [![Gitpod Ready-to-Code](https://img.
 
 Create a file named `your-username.json` in the directory `public/data` with the following content:
 
-Optional fields: `links` and `milestones`
+Optional fields: `links`,`milestones` and `type(personal | community)`
```
Comment (Panquesito7) [inline README.md:56]: I think the `type` field is obligatory, not sure, though. @eddiejaoude?

Comment (Vyvy-vi) [reply README.md:56]: It's supped to be optional

Comment (Vyvy-vi) [reply README.md:56]: and defaults to personal

**Review by Vyvy-vi — APPROVED** [review]
looks good to me

File: README.md (hunk 1 above)
Comment (Panquesito7) [inline README.md:56]:
```suggestion
Optional fields: `links`, `milestones`, and `type(personal | community)`
```

Comment (krishguptadev) [reply README.md:56]: wrong. comma appears after `and`

Comment (krishguptadev) [inline README.md:56]:
```suggestion
Optional fields: `links`, `milestones` and, `type(personal | community)`
```

Comment (krishguptadev) [reply README.md:56]: Superseded by https://github.com/EddieHubCommunity/LinkFree/pull/821/files#r771081622

Comment (Panquesito7) [reply README.md:56]: That's wrong according to Grammarly.

Comment (krishguptadev) [reply README.md:56]: Hmm, my Grammarly asks me to keep it after

image

Comment (nhcarrigan) [reply README.md:56]: This is incorrect, the Oxford comma follows the second to last item in the list and precedes the `and`.

```suggestion
Optional fields: `links`, `milestones`, and `type(personal | community)`
```

Comment (krishguptadev) [reply README.md:56]: So Grammarly is incorrect? 🤔

Comment (nhcarrigan) [reply README.md:56]: Must be a bug. Even [their article](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) shows it as I've indicated.

Comment (emmalearnscode) [reply README.md:56]: Oxford comma precedes the 'and' but is also entirely optional in English. Sometimes it helps the reader understand but in this case it's really not necessary.

Comment (krishguptadev) [reply README.md:56]:
```suggestion
Optional fields: `links`, `milestones` and `type(personal | community)`
```

**Review by emmalearnscode — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Vyvy-vi) [reply README.md:56]: I think relying on Grammarly isn't a good idea(except for spelling checks)

Comment (krishguptadev) [reply README.md:56]: We still need the space.

Comment (krishguptadev) [inline README.md:56]:
```suggestion
Optional fields: `links`, `milestones` and `type(personal | community)`
```

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**